_site/posts/2022-09-16_arrow-and-rpy2/index.html

```diff
@@ -0,0 +1,1102 @@
+<span id="cb13-4"><a href="#cb13-4" aria-hidden="true" tabindex="-1"></a>fiction2</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
+<div class="cell-output cell-output-display" data-execution_count="7">
+<pre><code>pyarrow.Table
+Trove ID: int64
+Common Title: string
+Publication Title: string
+Start Date: string
+End Date: string
+Additional Info: string
+Length: double
+Curated Dataset: string
+Identified Sources: string
+Publication Source: string
+Newspaper ID: int64
+Newspaper: string
+Newspaper Common Title: string
+Newspaper Location: string
+Newspaper Type: string
+Colony/State: string
+Author ID: int64
+Author: string
+Other Names: string
+Publication Author: string
+Gender: string
+Nationality: string
+Nationality Details: string
+Author Details: string
+Inscribed Gender: string
+Inscribed Nationality: string
+Signature: string
+Name Category : string
+----
+Trove ID: [[1,2,3,4,5,...,35491,35492,35493,35494,35495]]
+Common Title: [["The Mystery of Edwin Drood","The Mystery of Edwin Drood","Sporting Recollections in Various Countries","Brownie's Triumph","The Forsaken Bride",...,"The Heart of Maureen","His Lawful Wife","Love's Reward","Only a Flirt","The Doctor's Protegee"]]
+Publication Title: [["The Mystery of Edwin Drood","The Mystery of Edwin Drood","Sporting Recollections in Various Countries","The Jewels","Abandoned",...,"The Heart of Maureen","His Lawful Wife","Love's Reward","Only a Flirt","The Doctor's Protegee"]]
+Start Date: [["1871-03-04","1871-03-07","1847-06-16","1880-05-08","1880-08-21",...,"1914-01-06","1912-10-26","1911-02-04","1916-05-06","1911-11-25"]]
+End Date: [["1871-06-03","1871-05-16","1847-07-07","1880-08-14","1880-12-18",...,"1914-01-06","1912-10-26","1911-02-04","1916-05-06","1911-11-25"]]
+Additional Info: [[null,null,null,null,"Fiction. From English, American and Other Periodicals",...,"Published by special arrangement. All rights reserved.","Published by special arrangement. All rights reserved.","Published by special arrangement. All rights reserved.","All  Rights Reserved","Published by special arrangement. All rights reserved."]]
+Length: [[0,0,0,0,0,...,0,0,0,0,0]]
+Curated Dataset: [["Y","Y","Y","Y","Y",...,"N","N","N","N","N"]]
+Identified Sources: [["LCVF","LCVF","WPEDIA","TJW","TJW",...,null,null,null,null,null]]
+Publication Source: [[null,null,"Sunday Times",null,null,...,null,null,null,null,null]]
+...</code></pre>
+</div>
+</div>
+<p><br></p>
+<p>The <code>fiction2</code> object contains the same data as <code>fiction</code> but it is structured as an Arrow Table, and the data is stored in memory allocated to Arrow. Python itself only stores some metadata and the C++ pointer that refers to the Arrow Table. This isn’t exciting, but it will be important (and powerful!) later in a moment we transfer the data to R.</p>
```

Comment:
Maybe "by" Arrow instead of "to" Arrow? As is, I think the point that Arrow's doing the heavy lifting is actually affected a it by the choice of word there.

Reply:
Ah good catch. Thanks!